libs/wire-api/test/unit/Test/Wire/API/Routes/Version.hs

```diff
@@ -1,21 +1,44 @@
 module Test.Wire.API.Routes.Version where
 
+import Data.Aeson as Aeson
+import Data.Binary.Builder
+import Data.ByteString.Conversion
+import Data.String.Conversions (cs)
 import Imports
-import qualified Test.Tasty as T
+import Servant.API
+import Test.Tasty
 import Test.Tasty.HUnit
 import Wire.API.Routes.Version
 
-tests :: T.TestTree
+tests :: TestTree
 tests =
-  T.testGroup "Version" $
-    [ T.testGroup
-        "toPathComponent"
-        [testCase "serialise different versions" testToPathComponent]
+  testGroup
+    "Version always has the shape V<int>; serializations of Version and VersionNumber are `v<int>`, `<int>`, resp.  <int> is non-negative."
+    [ testCase "Version, show, 'v' prefix" $ do
```

Comment:
```suggestion
    [ testCase "Version, show, 'v' prefix" $ 
```

Is the `do` needed? (Please be careful with applying my suggestion: I haven't tested it. :warning: )

Reply:
it's not there because it needs to be, but because i want it to :)